components/automate-ui/src/app/modules/infra-proxy/client-details/client-details.component.html

```diff
@@ -32,8 +32,11 @@
         </chef-tab-selector>
       </chef-page-header>
 
-      <app-reset-client-key [openEvent]="openNotificationModal" 
-      [serverId]="serverId" [orgId]="orgId" [name]="name"></app-reset-client-key>
+      <app-reset-client-key
+        [openEvent]="openNotificationModal"
+        [serverId]="serverId"
+        [orgId]="orgId"
+        [name]="name"></app-reset-client-key>
```

Comment:
Author note: each attribute on a separate line make this easier to comprehend instead of one (hidden by itself), then three on the next line.